### PR TITLE
Loot crates no longer explode after unlocking

### LINKED ISSUE #36206: Abandoned crates instantly explode when clicked on
[Round ID]: # 84562
testmerged:
#33166 
#36029 
#36175 

empty hand used, clicked on an abandoned crate, blew up instantly. multitool tells me:

DECA-CODE LOCK REPORT:

* Anti-Tamper Bomb will activate after 10 failed access attempts.

but using an empty hand nevertheless immediately blows it up.

### PR DESCRIPTION
**Pull request by Time-Green**
Closes #36206

:cl:
tweak: loot crates can't explode after unlocking anymore
/:cl:
Quality of life. Basicly after unlocking a loot crate people sometimes still hit it with a multitool and now that it's unlocked it doesn't intercept, so it explodes and fucking kills you

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
I feel your pain.

**Comment by ShizCalev** [comment]
Fixes #36206?

**Comment by Time-Green** [comment]
@ShizCalev it doesn't. To be honest judging from code, what the guy described should be impossible and he probably just hit it with his multitool

**Comment by Time-Green** [comment]
speedmerge. i did it again

**Comment by Therapist1** [comment]
this has happened to me as well. ouch

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm considering this a fix even if it doesn't fix an issue on the tracker.

This sounds like an oversight.

**Review by ExcessiveUseOfCobblestone — DISMISSED** [review]

**Comment by davethwave** [comment]
I once scanned one once via the cargo scan tool thing and it went boom...

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -169,6 +169,7 @@
 				locked = FALSE
 				cut_overlays()
 				add_overlay("securecrateg")
+				tamperproof = 0
```
Comment (ShizCalev) [inline code/modules/mining/abandoned_crates.dm]: would you mind putting a comment for why that's there so folks know at a glance what this memery is

Comment (Time-Green) [reply code/modules/mining/abandoned_crates.dm]: Sure

**Review by ShizCalev — APPROVED** [review]

**Comment by knacker48** [comment]
I once used a genetic sequencer scanner by on it by accident and it exploded

**Pull request merged**